src/stock_str.rs

```diff
@@ -335,6 +335,11 @@ pub enum StockMessage {
 
     #[strum(props(fallback = "%1$s changed their address from %2$s to %3$s"))]
     AeapAddrChanged = 122,
+
+    #[strum(props(
+        fallback = "You changed your email address.\n\nContacts using Delta Chat will automatically replace you in the groups you send a message to.\n\nIf possible, set up your old email provider to forward all emails to your new email address."
```

Comment:
It's longer but also a very rare event. So it's ok to be more verbose IMO. 
```suggestion
        fallback = "You changed your email address from %1$s to $2$s.\n\nIf you now send a message to a chat, contacts there will automatically replace the old with your new address.\n\nIt's highly advised to set up your old email provider to forward all emails to your new email address. Otherwise you might miss messages of contacts who did not see messages after you changed your address."
```

Reply:
i think, without further background to the current limitations, the second one is hard to understand. so, i'd use the first one, maybe even a bit broader:

> Otherwise you might miss messages of contacts who did not get your new address yet.

but i see one could argue differently, i would be fine with all three variations, this is no blocker.